Adapters/Vungle/ISVungleAdapter/ISVungleBannerAdapterRouter.m

```diff
@@ -0,0 +1,195 @@
+//
+//  ISVungleBannerAdapterRouter.m
+//  ISVungleAdapter
+//
+//  Copyright © 2020 ironSource. All rights reserved.
+//
+
+#import "ISVungleBannerAdapterRouter.h"
+#import "ISVungleAdapter.h"
+
+@interface ISVungleBannerAdapterRouter ()
+
+@property (nonatomic, strong, nullable) NSString *bidPayload;
+
+@end
+
+@implementation ISVungleBannerAdapterRouter
+
+- (instancetype)initWithPlacementID:(NSString *)placementID
+                           delegate:(id<ISBannerAdapterDelegate>)delegate {
+    if (self = [super init]) {
+        _placementID = placementID;
+        _delegate = delegate;
+        _bannerState = UNKNOWN;
+        _bidPayload = nil;
+        _bannerSize = BannerSizeRegular;
+    }
+
+    return self;
+}
+
+- (void)loadBannerAd {
```

Comment:
since u removed dismissBannerWithServerData from Adapter class i suggest u call the destroy method if State == SHOWING in at line 33. 
IS wont create a new Banner instanse if their refresh mechanism is on. SO just to be safe its better we destroy any playing instances here. @YueVungle

Reply:
@ManojBudumuru-vungle Hi Manoj, sorry for not describing clearly, through debugging, I found IS will create a new Banner instance if their refresh mechanism is on, below is the debug：

Reply:
Ok. Thanks Yueting. Since a new instance is always created, then destroy doesn't need to be called.